src/main/java/com/ericsson/ei/controller/FailedNotificationControllerImpl.java

```diff
@@ -35,29 +30,33 @@
 import com.ericsson.ei.utils.ResponseMessage;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
 /**
- * This class represents the REST GET mechanism to extract the aggregated data on the basis of the SubscriptionName from the Missed Notification
- * Object.
+ * This class represents the REST GET mechanism to extract the aggregated data
+ * from the Failed Notification Object for a certain SubscriptionName.
```

Comment:
Can you possibly rephrase this sentence into something more human readable? How about: "This class contains logic for retrieving failed notifications for the given subscription."

Reply:
Well, I am quite sure that humans can read this.... However, I changed it with your suggestion.